README.md

```diff
@@ -20,6 +20,10 @@ require "browser"
 
 browser = Browser.new("Some User Agent", accept_language: "en-us")
 
+# if using Rack based apps (.e.g Rails, Sinatra), add this to your controller:
+
+browser = Browser.new(request.user_agent, accept_language: "en-us")
```

Comment:
Fwiw, Rails is integrated automatically, so you can you use `browser` from both controllers and views without having to instantiate the object first.

https://github.com/fnando/browser#rails-integration

Reply:
very good! I did not realise this.